components/secondary_button/style.css

```diff
@@ -0,0 +1,27 @@
+.BC-Gov-SecondaryButton {
```

Comment:
This should probably be confirmed with a developer, but I think you should specify "background: none"; I think if you don't there's a chance that a default styling or other CSS styling could apply a background to it.